install/authentik-install.sh

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: remz1337
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies (Patience)"
+$STD apt-get install -y {curl,sudo,mc}
+$STD apt-get install -y gpg pkg-config libffi-dev
+$STD apt-get install -y --no-install-recommends build-essential libpq-dev libkrb5-dev
+$STD apt-get install -y libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev pkg-config libffi-dev zlib1g-dev libxmlsec1 libxmlsec1-dev libxmlsec1-openssl libmaxminddb0
+msg_ok "Installed Dependencies"
+
+msg_info "Installing yq"
+YQ_LATEST="$(wget -qO- "https://api.github.com/repos/mikefarah/yq/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")')"
+$STD wget "https://github.com/mikefarah/yq/releases/download/${YQ_LATEST}/yq_linux_amd64" -qO /usr/bin/yq
+chmod +x /usr/bin/yq
+msg_ok "Installed yq"
+
+msg_info "Installing Python 3.12"
+wget -q https://www.python.org/ftp/python/3.12.1/Python-3.12.1.tgz -O Python.tgz
+tar -zxf Python.tgz
+cd Python-3.12.1
+$STD ./configure --enable-optimizations
+$STD make altinstall
+$STD cd -
+$STD rm -rf Python-3.12.1
+$STD rm -rf Python.tgz
+$STD update-alternatives --install /usr/bin/python3 python3 /usr/local/bin/python3.12 1
+msg_ok "Installed Python 3.12"
+
+NODE_VER="22"
```

Comment:
Does it have to be the 22, or can we also use the standard NodeJS from Debian? In the end, that was only 3 lines instead of 8 or 9

Reply:
Might be worth a try, but this step either it be 4 or 8 line doesn't take more/less time. NodeJS v22 is what Authentik uses in it's Docker image so I think it's best to stick with the same dependencies and avoid potential compatibility issues